samples/watchos/plists/App/Info.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
```

Comment:
Should this file be commited? Isn't it generated by `xcodegen`?

Reply:
Added, as agreed to not force people use it.